docs/src/concepts/bc.md

```diff
@@ -42,5 +61,5 @@ The following figure showcases a 2D square domain $\Omega$ with different bounda
 To launch a kernel that satisfies these boundary conditions in Chmy.jl, you can use the following code:
 
 ```julia
-bc!(arch, grid, field => (x = (Dirichlet(a), Dirichlet(b)), y = Neumann()))
+bc!(arch, grid, field => (y = (Dirichlet(a), Dirichlet(b)), x = Neumann()))
```

Comment:
```suggestion
bc!(arch, grid, field => (x = Neumann(), y = (Dirichlet(b), Dirichlet(a))))
```

Reply:
I feel it's less confusing to first define `x` BCs, the `y` BCs. Also, in `y` direction, the "bottom" is the first one, the "top" the second one.

Reply:
Thank you! Indeed I recall Ivan said it's always from "left" to "right" regarding the ordering of directions that shall appear